docker/data-science/notebooks/ch05/3.semantic-knowledge-graph.ipynb

```diff
@@ -715,49 +682,26 @@
       "chef  0.80689\n",
```

Comment:
These results look bad and different from what's in the manuscript. I'll run this notebook locally to see what's going on.

Reply:
Solved with min_popularity being added